includes/class-sensei-modules.php

```diff
@@ -1243,28 +1243,69 @@ public function course_columns( $columns = array() ) {
 	 */
 	public function course_column_content( $column = '', $course_id = 0 ) {
 		if ( 'modules' === $column ) {
-			$modules = $this->get_course_modules( $course_id );
-
-			if ( $modules ) {
-				$module_links = [];
-				foreach ( $modules as $module ) {
-					$module_links[] = sprintf(
-						'<a href="%s">%s</a>',
-						esc_attr( get_edit_term_link( $module->term_id ) ),
-						esc_html( $module->name )
-					);
-				}
+			$this->output_course_modules_column( $course_id );
+		}
+	}
 
-				echo implode( ', ', $module_links ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Output is escaped in the array.
+	/**
+	 * Output the course modules column HTML.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @param int $course_id
+	 * @return void
+	 */
+	private function output_course_modules_column( int $course_id ) {
+		$modules = $this->get_course_modules( $course_id );
 
-				// Output the edit modules order link.
-				echo sprintf(
-					'<a class="button-link" href="%s">%s</a>',
-					esc_url( admin_url( 'options.php?post_type=course&page=module-order&course_id=' . intval( $course_id ) ) ),
-					esc_html__( 'Order modules', 'sensei-lms' )
-				);
+		if ( ! $modules ) {
+			return;
+		}
+
+		/**
+		 * Filter to change the number of links in the course modules column.
+		 *
+		 * @since 4.0.0
+		 * @hook  sensei_modules_column_max_links_count
+		 *
+		 * @param  {int} $max_links_count The number of links.
+		 * @return {int}
+		 */
+		$max_links_count = apply_filters( 'sensei_modules_column_max_links_count', 3 );
```

Comment:
Most of the other filter names in this file start with `sensei_module`, so we should probably use the singular form here too. In order to distinguish it from the module column on the _Lessons_ page, we might even want to call it something like `sensei_module_course_column_max_links_count`? Naming is hard. 😅

Reply:
Naming is definitely hard, so thanks for coming up with the new name! Updated in 4c75316b861c6fe3c0a0446d8927e901ee9d06ee.